express-static/server.js

```diff
@@ -0,0 +1,7 @@
+import express from 'express';
+
+const app = express();
+
+app.use(express.static('public'));
+
+app.listen(8080, console.log('Reporting to you live from Port 8080!'));
```

Comment:
The `console.log` should be wrapped inside a callback function, otherwise it runs before the app starts listening